pubspec.yaml

```
@@ -20,4 +21,7 @@ dev_dependencies:
   flutter_lints: ^4.0.0
 
 flutter:
-  uses-material-design: true
\ No newline at end of file
+  uses-material-design: true
+
+  assets:
+    - assets/images/
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)